src/Components/Samples/BlazorServerApp/Pages/_Host.cshtml

```diff
@@ -2,6 +2,10 @@
 @namespace BlazorServerApp.Pages
 @addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
 
+@{
+    ViewContext.DisableAutomaticComponentPersistence(); 
```

Comment:
By default the state is save automatically if you are rendering a single component on the page. You can disable this behavior on a per page basis if you are rendering multiple components or "globally" using a _viewstart.cshtml for pages in a given folder or the entire app